net-misc/tigervnc/files/tigervnc-1.12.0.initd

```diff
@@ -59,10 +55,10 @@ stop() {
 		# vncserver no longer provides a `-kill` option
 		# killing vncsession does not work, we have to kill Xvnc
 		# run pstree to see exactly what's going on
-		sessionpid=`cat /run/vncsession-\:${user##*:}.pid`
-		serverpid=`pstree -p $sessionpid | grep Xvnc | sed -e 's/^.*Xvnc(//
-			s/).*$//'`
-		kill $serverpid &>/dev/null
+		sessionpid=$(cat /run/vncsession-:"${user##*:}".pid)
+		serverpid=$(pstree -p "$sessionpid" | grep Xvnc | sed -e 's/^.*Xvnc(//
+			s/).*$//')
+		kill "$serverpid" >/dev/null 2>&1
```

Comment:
I guess sessionpid and serverpid should never contain whitespace or special characters. Thus, the quotations marks seem superfluous to me.

Reply:
Yes, I added them only to silence `shellcheck`.  They don't do any harm anyway.

Reply:
Overquoting is generally better than underquoting, yep.